### DISCUSSION TITLE
for population of CKD stages in version 10, Did they take into account the CYP3A4 expression level variability in CKD?

### DISCUSSION THREAD

**Discussion by sajabzughoul** [Open-Systems-Pharmacology/Forum#1045] [Questions & Problems]
I want to ask about a Populations for CKD stages in PK-Sim version 10, did they account for CYP3A4 level variability published in literature? this paper includes the decrease in expression ''Modeling and predicting drug
pharmacokinetics in patients
with renal impairment
Expert Rev. Clin. Pharmacol. 4(2), 261–274 (2011)''
it seems that they did not account for that because the concentration in all populations was 4.3 micromole/l, and I cannot edit that even in simulation.

**Comment by prvmalik** [comment]
No decreases in nonrenal clearance are applied. The compounds in the evaluation were predominantly cleared renally. The population includes changes in foundational and systemic parameters that apply regardless of the route of elimination of the compound.

You can apply changes to nonrenal clearance manually as you see fit. To do this, make sure to set the appropriate reference concentration in the individual upon which your desired CKD population is templated. There is a ton of conflicting literature on the topic. One of my favourite methods is to use the mean changes parameterized by Sayama et al. 2014.

Also consider the impact on renal transporters. OCT transporter function is largely conserved, but OAT function may be severely decreased.

**Reply by sajabzughoul** [reply]
Did you mean that I cannot use the predefined templates for CKD stages ?

**Reply by prvmalik** [reply]
When you import a population template, it is based on an individual that you select. This individual defines the expression of enzymes and transporters. So if you want your CKD population to have a lower expression of enzymes or transporters, you must base it on an individual with these changes already applied. You can then apply variability (either default or user defined) on the following tab when constructing the population.

See the notes on the template when selecting it for import for more information

Pay close attention to the notation on fraction unbound.

An alternative approach would be to generate with healthy expression of enzymes and transporters, and then to export the population to csv and adjust manually as you see fit thereafter.

**Reply by WTY-tt** [reply]
Hello, would you like to consult the DOI number of the Sayama et al. 2014 you mentioned?

**Reply by prvmalik** [reply]
10.1208/s12248-014-9626-3 should be it

**Reply by WTY-tt** [reply]
Thank you very much!

**Comment by sajabzughoul** [comment]
Simulation using these templates does not confirm any edits, either default or user defined for CYP3A4 concentration 
please help me to use the templates and edits for CYP3A4 concentration

**Reply by prvmalik** [reply]
Make the edit in your individual building block first. So if you want 50% lower CYP3A4 then in your individual, change the reference concentration from 4.32 to 2.16. Then import/build the population based on this individual (when you select "population based on" individual).

**Reply by prvmalik** [reply]
Alternatively lower the Vmax or specific clearance by 50%, same result

**Reply by sajabzughoul** [reply]
Well done, thank you
